smoketest/scripts/cli/test_load_balancing_inbound-ipvsadm.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2020 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import jmespath
+import json
+import unittest
+
+from base_vyostest_shim import VyOSUnitTestSHIM
+
+from vyos.util import read_file
+
+base_path = ['load-balancing', 'inbound', 'ipvsadm']
+virtual_ip = '127.0.0.1'
+virtual_ip6 = 'fc00::1'
+IPVSADM_CONF = '/tmp/ipvsadm.rules'
+
+class TestLoadBalancingInbound(VyOSUnitTestSHIM.TestCase):
+    def setUp(self):
+        # ensure we can also run this test on a live system - so lets clean
+        # out the current configuration :)
+        self.cli_delete(base_path)
+
+    def tearDown(self):
+        self.cli_delete(base_path)
```

Comment:
If `ipvsadm` uses a process, test this is `process_named_running()`. Also after deletion, check that the process no longer runs.

Reply:
No, it's not a background service. Ipvsadm is the user program of Linux virtual server, which is a kernel feature